### ISSUE TITLE
Additional Contributors proposal

### ISSUE THREAD

**Issue by morg0n** [hackforblacklivesmatter-5-1/rightshub#36]
Hey all,  I accidentally found this repo and am extremely impressed and motivated by the work you are doing.  I was wondering if you were looking for additional contributors; myself and potentially one other engineer (she specializes in design) would absolutely love the opportunity to work on this.

**Comment by skut21x-ga** [comment]
Hi there! Let me consult with the rest of this team. We made it for a hackathon that has since ended - though we did plan on continuing to add more features to it. Did you all have anything in mind specifically you wanted to contribute?

**Comment by morg0n** [comment]
Do you all have a security engineer on board?  
Would the interpretation be correct in that the list of people is an HTML contain being controlled with JavaScript, querying either a database or json (load json every 1-2 hours from database to save on traffic bandwidth and strain)? 
Either way, if there is room on the team I'd love to contribute, and if not, no worries :)

**Comment by skut21x-ga** [comment]
Yes - The website was created via React using JavaScript/JSX. The data JSON is static and local as there currently is no existing API or live records provided for this data that we could find. We converted it from a CSV provided by another website. If you have ideas for functionality or design aside from the JSON, I can speak to the team. I know we had discussed D3 and visualizations, but were short on time.
